### ISSUE TITLE
[Enhancement]: Strimzi Metrics Reporter support

### ISSUE THREAD

**Issue by OwenCorrigan76** [strimzi/strimzi-kafka-operator#10753]
### Related problem

[Strimzi Metrics Reporter ](https://github.com/strimzi/metrics-reporter/releases/tag/0.1.0-rc3) is a new project under the Strimzi organization that directly exposes metrics via an HTTP endpoint in the Prometheus format. We should add support for it to Strimzi. The motivation for creating the new Metrics Reporter can be found in this [proposal](https://github.com/strimzi/proposals/blob/main/064-prometheus-metrics-reporter.md).

### Suggested solution

There are several stages that would need to be added for this enhancement, including:

- Add integration to Kafka broker and controllers
- Add integration to KafkaConnect and MirrorMaker2
- Add integration to the Bridge

**Comment by im-konge** [comment]
Triaged on 31.10.2024: This makes sense to implement.

**Comment by smilingprogrammer** [comment]
Hi everyone. I am new to the strimzi community.

I want to start contributing to the strimzi project. Is this a fair open issue i can start with?

**Comment by scholzj** [comment]
No, sorry, I think @OwenCorrigan76 is working on this already. Best is to look for issues with the `help wanted` or `good start` labels.

**Comment by scholzj** [comment]
@OwenCorrigan76 @fvaleri I guess the docs for the Bridge are still missing from this issue?

**Comment by fvaleri** [comment]
@scholzj I think Owen is on vacation, but he was working on that and will open a PR when he is back.

**Comment by OwenCorrigan76** [comment]
@scholzj @fvaleri I'll open a PR for the docs over the next couple of days. Thanks

**Comment by OwenCorrigan76** [comment]
PR Opened for docs update: https://github.com/strimzi/strimzi-kafka-operator/pull/11727
